pkg/apis/config/v1beta1/types.go

```diff
@@ -73,3 +74,20 @@ type CapacitySchedulingArgs struct {
 	// KubeConfigPath is the path of kubeconfig.
 	KubeConfigPath *string `json:"kubeconfigpath,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// TargetLoadPackingArgs holds arguments used to configure TargetLoadPacking plugin.
+type TargetLoadPackingArgs struct {
+	metav1.TypeMeta 	`json:",inline"`
+
+	// Node target CPU Utilisation for bin packing
+	TargetCPUUtilization float64 `json:"mode,omitempty"`
+	// Default CPU requests to use for best effort QoS
+	DefaultCPURequests   int64 `json:"mode,omitempty"`
+}
+
+// TODO(aqadeer): Use codegen for this
+func (b TargetLoadPackingArgs) DeepCopyObject() runtime.Object {
+	panic("implement me")
```

Comment:
IMO, you may generate the codes first

Reply:
Wasn't able to due to https://github.com/kubernetes-sigs/scheduler-plugins/issues/116. It's resolved now and I updated.